src/scgpt/cell_type_annotation/config.vsh.yaml

```diff
@@ -0,0 +1,156 @@
+functionality:
+  name: cell_type_annotation
+  namespace: "scgpt"
+  description: |
+    Annotate adata with cell types through the scGPT model.
+  authors:
+    - __merge__: /src/authors/dorien_roosen.yaml
+      roles: [ maintainer, author ]
+    - __merge__: /src/authors/jakub_majercik.yaml
+      roles: [ maintainer, author ]
+
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: "--input"
+          type: file
+          direction: input
+          required: true
+          example: resources_test/scgpt/test_resources/Kim2020_Lung_preprocessed.h5mu
+          description: |
```

Comment:
Could you be a little bit more specific about what preprocessing steps are required?